frontend/packages/console-shared/src/components/dynamic-form/utils.ts

```diff
@@ -13,14 +13,6 @@ export const getSchemaErrors = (schema: JSONSchema6): SchemaError[] => {
           },
         ]
       : []),
-    ...(_.isArray(schema.type)
```

Comment:
Swagger OpenAPI schema doesn't allow array types, so this check was not necessary.